core/platform/email/gae_email_services.py

```diff
@@ -22,6 +22,15 @@
 
 
 def get_incoming_email_address(reply_to_id):
+    """Returns string detailing the id of the reply-client.
+
+    Args:
+      - reply_to_id: str or None. The unique reply-to id used in reply-to email
```

Comment:
Indent by 4 (remove `-` from the beginning as you have done below).

Reply:
Done